client/src/Components/Search/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { SearchOutlined } from '@ant-design/icons';
+import Button from '../Button';
+import Select from '../Select';
+
+import './style.css';
+
+import { locations, serviceTypes } from '../../Utils/data';
+
+const Filter = ({ handleSearch }) => {
+  const [service, setService] = useState('null');
+  const [location, setLocation] = useState('null');
+
+  handleSearch(service, location);
+
+  const handleService = (value) => {
+    setService(value);
+  };
+  const handleLocation = (value) => {
+    setLocation(value);
+  };
```

Comment:
* This component is the presentational component that doesn't contain state, it just uses the function from props to set these input value from the parent component -- remove all this code
* you can use setService and SetLocation functions directly, why you put them inside a function?
```
// props
const Filter = ({ handleSearch, setService , setLocation }) //these functions pas from parent component
```